### PR TITLE
[i18n-<languageCode>] Translating Benchmarks.md to Chinese

### PR DESCRIPTION
**Pull request by asdkfjsd**
# What does this PR do?

I have translated Benchmarks.md to Chinese.  Please let me know if there are any corrections. Thank you very much!
According to the context, I translated  'Benchmark best practices' as 'Recommended Strategies for Benchmarking' in Chinese .

issue: [i18n-<languageCode>] Translating Benchmarks.md to Chinese #35134
Fixes # [https://github.com/huggingface/transformers/issues/35134](url)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by Rocketknight1** [comment]
cc @stevhliu

**Review by stevhliu — COMMENTED** [review]
Thanks for the translation. Overall, I think it looks good, but its just a little difficult to read with both English/Chinese content.

Can you also find another contributor to review the translated content (I can't read Chinese)?

File: docs/source/zh/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+
+让我们来看看如何对🤗 Transformers模型进行基准测试，以及进行测试的推荐策略和已有的基准测试结果。
+
+如果你需要更详细的回答，可以在这里找到关于如何对🤗 Transformers模型进行基准测试的[说明](https://github.com/huggingface/notebooks/tree/main/examples/benchmark.ipynb)。
+
+<!-- Let's take a look at how 🤗 Transformers models can be benchmarked, best practices, and already available benchmarks.
+
+A notebook explaining in more detail how to benchmark 🤗 Transformers models can be found [here](https://github.com/huggingface/notebooks/tree/main/examples/benchmark.ipynb). -->
+
+<!-- ## How to benchmark 🤗 Transformers models -->
```
[the remark this answers is not in the document]
Comment (asdkfjsd) [reply docs/source/zh/benchmarks.md:36]: Thank you for your advice. I will make improvements soon.

**Comment by asdkfjsd** [comment]
I have removed all the English original text (which was previously kept as comments in the document) and refined some of the Chinese expressions. Any suggestions or corrections are welcome.

**Comment by HMJ0628** [comment]
The translation is good and the meaning is clear.

**Review by stevhliu — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**